events/src/main/kotlin/edu/artic/events/AllEventsFragment.kt

```diff
@@ -0,0 +1,51 @@
+package edu.artic.events
+
+import android.os.Bundle
+import android.support.v7.widget.GridLayoutManager
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.View
+import com.fuzz.rx.bindToMain
+import com.fuzz.rx.disposedBy
+import edu.artic.adapter.itemChanges
+import edu.artic.events.recyclerview.AllEventsItemDecoration
+import edu.artic.viewmodel.BaseViewModelFragment
+import kotlinx.android.synthetic.main.fragment_all_events.*
+import kotlin.reflect.KClass
+
+class AllEventsFragment : BaseViewModelFragment<AllEventsViewModel>() {
+
+    override val viewModelClass: KClass<AllEventsViewModel>
+        get() = AllEventsViewModel::class
+    override val layoutResId: Int
+        get() = R.layout.fragment_all_events
+    override val title: String
+        get() = "Events" // TODO: add to strings or figure out language stuff
```

Comment:
I have added all the strings requiring localizations to string resource (base module) file so that all the strings are at same place.